packages/mds-policy/api.ts

```diff
@@ -64,84 +65,105 @@ function api(app: express.Express): express.Express {
     next()
   })
 
-  app.get(pathsFor('/policies'), async (req: PolicyApiRequest, res: GetPoliciesResponse) => {
-    // TODO extract start/end applicability
-    // TODO filter by start/end applicability
-    const { start_date = now(), end_date = now() } = req.query
-    const { scopes } = res.locals
-
-    /*
-      If the client is scoped to read unpublished policies,
-      they are permitted to query for both published and unpublished policies.
-      Otherwise, they can only read published.
-    */
-    const { get_published = null, get_unpublished = null } = scopes.includes('policies:read')
-      ? parseRequest(req, { parser: x => (x ? JSON.parse(x) : null) }).query('get_published', 'get_unpublished')
-      : { get_published: true }
-
-    if (start_date > end_date) {
-      res.status(400).send({ error: 'start_date after end_date' })
-      return
-    }
+  app.get(
+    pathsFor('/policies'),
+    async (req: PolicyApiRequest, res: GetPoliciesResponse, next: express.NextFunction) => {
+      // TODO extract start/end applicability
```

Comment:
Can these TODOs be deleted? It looks like they've been done.

Reply:
Yeup